packages/wrangler/src/__tests__/configuration.test.ts

```diff
@@ -40,6 +40,10 @@ describe("normalizeAndValidateConfig()", () => {
 			tsconfig: undefined,
 			kv_namespaces: [],
 			legacy_env: true,
+			logfwdr: {
+				bindings: [],
+				schema: "",
```

Comment:
should default here be undefined?
```suggestion
				schema: undefined,
```